scripts/mock_icd_generator.py

```diff
@@ -765,7 +769,7 @@
 ''',
 'vkGetPhysicalDeviceProperties': '''
     // TODO: Just hard-coding some values for now
-    pProperties->apiVersion = VK_API_VERSION_1_0;
+    pProperties->apiVersion = VK_API_VERSION_1_1;
```

Comment:
Might be more robust to use a single local constant for the version, rather than 3 references to the VK constant.

Reply:
Thanks, Mike. I'll land this and submit a follow-on to converge all the version refs.